JoinColony/colonyCDapp
# feat: add staged payment description

## Description

- Added description to staged payment
- I had to modify a few components to retrieve the recipient's data

## Testing

* Install staged payment extension
* Create action with one and many token addresses to check both descriptions

Resolves https://github.com/JoinColony/colonyCDapp/issues/3305
